dashboard/lib/service/dev_cocoon.dart

```diff
@@ -135,8 +135,8 @@ class DevelopmentCocoonService implements CocoonService {
   }
 
   @override
-  Future<bool> rerunTask(Task task, String accessToken) async {
-    return false;
+  Future<CocoonResponse<bool>> rerunTask(Task task, String accessToken) async {
+    return const CocoonResponse<bool>.error('view errorBrook effect from dev cocoon');
```

Comment:
I don't understand this error message.

Why not just `Unable to retry against fake data. Try building the app to use prod data.`